types/index.test.ts

```diff
@@ -88,7 +142,45 @@ const returnWithContextDone: AzureFunction = function (context: Context, _req: H
     context.done(null, { myOutput: { text: 'hello there, world', noNumber: true } });
 };
 
-export { runHttp, cookieFunction, runHttpReturn, runServiceBus, runFunction, runHttpWithQueue, returnWithContextDone };
+const returnWithSend: AzureFunction = function (context: Context, req: HttpRequest) {
+    context.res = context.res as HttpResponseApi;
+    context.res.send('hello world');
+};
+
+const returnWithEnd: AzureFunction = function (context: Context, req: HttpRequest) {
+    context.res = context.res as HttpResponseApi;
+    context.res.end('hello world');
```

Comment:
The point of this file is to verify the types, not run any of the code. So you could just move these `end`, `send`, etc. calls into one function instead of separating it out like this

Reply:
Makes sense